parsers/ES.py

```diff
@@ -140,17 +139,15 @@ def fetch_consumption(
     target_datetime: Optional[datetime] = None,
     logger: Logger = getLogger(__name__),
 ) -> List[dict]:
-    check_valid_parameters(zone_key, session, target_datetime, logger)
+    check_valid_parameters(zone_key, session, target_datetime)
 
     ses = session or Session()
     island_data = fetch_island_data(zone_key, ses, target_datetime)
     consumption = TotalConsumptionList(logger)
     for event in island_data:
         consumption.append(
             zoneKey=zone_key,
-            datetime=datetime.utcfromtimestamp(event.timestamp).astimezone(
-                timezone.utc
-            ),
+            datetime=datetime.fromtimestamp(event.timestamp).astimezone(timezone.utc),
```

Comment:
I noticed I made a error in the previous PR and corrected for UTC twice. This should fix that and I'll re-fetch the consumption and exchange data once this is merged.